pypesto/ensemble/ensemble.py

```diff
@@ -1204,7 +1204,11 @@ def get_percentile_label(percentile: Union[float, int, str]) -> str:
     return f'{PERCENTILE} {percentile}'
 
 
-def calculate_cutoff(result: Result, percentile: float = 0.95):
+def calculate_cutoff(
+    result: Result,
+    percentile: float = 0.95,
+    cr_option: str = 'simultaneous',
```

Comment:
Add
```python
SIMULTANEOUS = 'simultaneous'
POINTWISE = 'pointwise
```
to C.py in the ensemble section [1] and use here.

[1]
https://github.com/ICB-DCM/pyPESTO/blob/622145db3fdf3c3e2747c34ba3e563fb34d58406/pypesto/C.py#L11

Alternatively, change to boolean argument `simultaneous=True` and document the meaning of `True` and `False` in the docstring

Reply:
Done.